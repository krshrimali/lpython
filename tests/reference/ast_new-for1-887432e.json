{
    "basename": "ast_new-for1-887432e",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/for1.py",
    "infile_hash": "f025995384e87a428a605ae718932f3fbc25aa3f8baa5508f2018997",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast_new-for1-887432e.stderr",
    "stderr_hash": "9a275949eb9a4410d7fb318974f62c4b1a70ed14bc01b9b886318220",
    "returncode": 1
}