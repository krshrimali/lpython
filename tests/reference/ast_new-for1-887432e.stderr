syntax error: Token ':' is unexpected here
 --> tests/parser/for1.py:4:16
  |
4 | for x in fruits:
  |                ^ 
